src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,37 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGHT_PASS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        for (User u : Storage.people) {
+            if (u.getLogin().equals(user.getLogin())) {
+                throw new RuntimeException("Login " + user.getLogin() + " has already exists");
+            }
+        }
```

Comment:
think about how to check is login present in DB. Do not use loop. Do not use direct access to Storage. Use dao.